launch/demo-components.launch.py

```diff
@@ -0,0 +1,42 @@
+import launch
```

Comment:
This file doesn't work, does it? If it doesn't, I don't think it should be committed in.

Reply:
I was just gonna leave this cause its more of joey's work with components if he wants to fix that at a later date

Reply:
Ah, okay. How about we add a comment to the top that says, like:
```
"""
DO NOT LAUNCH THIS FILE!

Due to CTRE diagnostic server issues, running multiple talons in the same process
(as is done through ComposableNode) causes all but one talon node to fail. Thus,
this launch file does not work. For now, please use `demo-nodes.launch.py` instead.
"""
```